javascript/modules/client/src/client-erc20.ts

```diff
@@ -189,20 +187,30 @@ export class ClientErc20 extends ClientCore implements IClientErc20 {
     // TODO: Unimplemented
     return Promise.resolve({ average: BigInt(0), max: BigInt(0) });
   }
+
+  //// PRIVATE RETRIEVAL HANDLERS
+  private _getMembers(daoAddressOrEns: string) {
+    if (!daoAddressOrEns) {
+      throw new Error("Invalid DAO address or ENS");
+    }
+
+    const mockAddresses = [
+      "0x8367dc645e31321CeF3EeD91a10a5b7077e21f70",
+      "0xDA9dfA130Df4dE4673b89022EE50ff26f6EA73Cf",
+      "0xBE0eB53F46cd790Cd13851d5EFf43D12404d33E8",
+      "0x2dB75d8404144CD5918815A44B8ac3f4DB2a7FAf",
+      "0xc1d60f584879f024299DA0F19Cdb47B931E35b53",
+    ];
+
+    return Promise.resolve(mockAddresses.filter(() => Math.random() > 0.4));
```

Comment:
Not strictly necessary, just FYI adding a small delay can help the UI check that a loading spinner appears while data is not available. Using a simple `setTimeout` could be of use. 

Example:

```ts
return new Promise(resolve => setTimeout(resolve, 1000)).then(() => mockAddresses.filter(() => Math.random() > 0.4));
```